scripts/setup-nat-gcloud-cluster.sh

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env bash
```

Comment:
The only difference is the docker image name, so why do we need a separate script for that although the image name is already a parameter?

Reply:
No, this needs to create the network and then start the other one bound to that network. This could be achieved by a separate image (as it is now) or use pure GCloud VM (not create-with-container). If that's what you mean?

Reply:
The network setup is handled in the init script of the container itself. Also the autmatically started nat nodes use the same `deploy.sh` scripts as the non-nat nodes. So I don't understand why we need this extra script.

Reply:
Ok, I see what you mean. Sure we do not need this script - the mere purpose of that is for testing and if someone wants to manually deploy just NAT'd nodes. At the same time, we could use the `start-gcloud-cluster.sh` with different parameters for that purpose. But with one change - we cannot use `wait_for_port 9091` for NAT nodes, since the 9091 port obviously cannot be exposed in their case.